torch/csrc/jit/profiling_graph_executor_impl.cpp

```diff
@@ -134,6 +134,12 @@ ExecutionPlan ProfilingGraphExecutorImpl::getPlanFor(Stack& stack) {
   EliminateDeadCode(copy);
   GRAPH_DUMP("Optimized Graph : ", copy);
   // cache
+
+  static const char* prof_opt = std::getenv("PYTORCH_OPT");
```

Comment:
This does not seem appropriate. I do not see it tested anywhere in this patch, seems like a leftover from development.

Reply:
will remove, I'm still using this for testing to make sure the correct executor is running